spec/order_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'order'
+
+describe Order do
+  # let(:menu_double)       { double :menu }
```

Comment:
Would be great to make use of the dependency injection you have set up in Order to use the Menu double. At the moment your tests aren't isolated. Please make use of the coaching team if you're not sure how to do this :)